### PR TITLE
New Security Page

### PR DESCRIPTION
**Pull request by jme783**
Includes documentation on impending "Quarantine" feature for products
- [x] Review text
- [ ] Dependent feature released

### CODE REVIEW

File: src/content/guide/how-to-build-a-product/security.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+# Security
+
+When it comes to IoT devices, there is no shortage of security concerns. This
+part of the guide will be a discussion of security-related topics that arise
+with a product at scale.
+
+## Treating Devices as Products
+
+When you created a product, you created a new type of device in the Particle
```
Comment (monkbroc) [inline src/content/guide/how-to-build-a-product/security.md:16]: You can link to the previous docs section `[When you created a product](/guide/how-to-build-a-product/dashboard/#defining-a-product)`

File: src/content/guide/how-to-build-a-product/security.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+cloud. So far, development kits that you have been using for prototyping have been considered as a "Photon," "P1", "Electron," or "Core" by the
+Particle cloud. However, when you added devices to your product, you are signaling to the cloud
+that you would like them to be treated as part of your product instead of
+as development kits.
+
+There are certain security implications that arise around devices behaving as products. Ideally, the following is always true:
+- Only the _devices_ that be given product privileges will recieve them
```
Comment (monkbroc) [inline src/content/guide/how-to-build-a-product/security.md:23]: This line is probably the result of some heavy editing cause I don't understand it ;)

File: src/content/guide/how-to-build-a-product/security.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+- Only the _people_ who should have the ability to interact with the
+  product device will be allowed to do so
```
Comment (monkbroc) [inline src/content/guide/how-to-build-a-product/security.md:25]: Only the people authorized to interact with the product device will be able to do so

File: src/content/guide/how-to-build-a-product/security.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+![A new product](/assets/images/product-id.png) <p class="caption">Your product
+ID is marked with a key icon</p>
+
+You should store your Product ID in a safe place where you won't forget it. It
+may be helpful to memorize your Product ID if possible.
+
+### Definitions of Product ID
+
+There are **2** places where a Product ID is defined for a given device.
```
Comment (monkbroc) [inline src/content/guide/how-to-build-a-product/security.md:49]: There are **2** places where a Product ID is stored for a given device.

(is a product id defined, stored, recorded or saved in a device/cloud? try to use consistent terms)

File: src/content/guide/how-to-build-a-product/security.md [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+
+A device that has been successfully added to a product will be granted special
+privileges. These privileges include (but are not limited to):
+
+- Receiving over-the-air (OTA) firmware updates from this product
+- Publishing events that will appear in this product's event stream
+- Claiming by a customer of your organization
+- Triggering product webhooks
+
+These privileges are only guaranteed when there are [no mismatches](#mismatched-product-ids) between the Product ID of the device in the cloud and the Product
```
Comment (monkbroc) [inline src/content/guide/how-to-build-a-product/security.md:86]: These privileges are only granted

**Comment by monkbroc** [comment]
This looks good. Nice explanation Jeff.

**Comment by jme783** [comment]
I am merging this in to `master` before the feature is released so that we can send out a notification email to product creators who will be affected by this change, and provide documentation for more info

**Pull request merged**